### ISSUE TITLE
SQLite blob problem

### ISSUE THREAD

**Issue by leonelfreire** [serge-rider/dbeaver#99]
Hi,

When I was trying to insert blobs in a SQLite database I noticed some strange behaviours.

First I tried with this DDL:

```
CREATE TABLE temp (
	mydata BLOB
);
```

And this happened:

![error1](https://cloud.githubusercontent.com/assets/14333/11702075/6b9e379a-9eb2-11e5-85e7-232ccb8a72f1.png)

I modified the DLL to:

```
CREATE TABLE temp2 (
	id INTEGER,
	mydata BLOB,
	CONSTRAINT TEMP2_PK PRIMARY KEY (id)
);
```

And apparently everything was ok:

![error2](https://cloud.githubusercontent.com/assets/14333/11702078/72c43c86-9eb2-11e5-9763-01071dd9a037.png)

But after a refresh the field was NULL:

![error3](https://cloud.githubusercontent.com/assets/14333/11702080/786ca916-9eb2-11e5-925c-3cc8bdefb8d7.png)

DBeaver version: 3.5.4 and 3.5.6.

**Comment by leonelfreire** [comment]
I had no problem using the CLI SQLite (version 3) to save blobs.

**Comment by leonelfreire** [comment]
Using MySQL + dbeaver I had no problem too.

**Comment by leonelfreire** [comment]
I made a small program in Java to insert a sample image:

https://github.com/leonelfreire/testsqlite

And the results:

![error4](https://cloud.githubusercontent.com/assets/14333/11703467/3d5f5a92-9ebe-11e5-9b81-390c6e142b5b.png)

But the same happens if I try to insert a new row using dbeaver:

![error5](https://cloud.githubusercontent.com/assets/14333/11703486/6954943c-9ebe-11e5-97fe-b70134949000.png)

**Comment by serge-rider** [comment]
It is a bug in SQLite driver + DBeaver misfunction.
SQLite driver silently does nothing when DBeaver tries to set BLOB value. As see from Xerial JDBC source code they added exceptions in non-supported API functions. So after next Xerial driver release this issue will be fixed automatically. Current Xerial version is 3.8.11.2 (https://github.com/xerial/sqlite-jdbc).
Additionally I'll fix LOB handler in DBeaver. Fix will be in 3.5.7 version.
Thanks for report.

**Comment by leonelfreire** [comment]
Thanks for your attention.
